vertical-pod-autoscaler/pkg/admission-controller/config.go

```diff
@@ -89,6 +89,22 @@ func selfRegistration(clientset *kubernetes.Clientset, caCert []byte, namespace,
 							Resources:   []string{"verticalpodautoscalers"},
 						},
 					},
+					{
+						Operations: []admissionregistration.OperationType{admissionregistration.Create, admissionregistration.Update},
```

Comment:
I might be missing something but I have the impression the `Update` is a bit too aggressive here.
Lets say that the user annotate the Deployment, this is an update, normally this does not create any rolling update, but it will! Same apply for Status update.

I don't see any code later that would prevent the update of the Deployment under such conditions.

Maybe we can just apply this to ReplicaSet Creation. If we do that we that would trigger resource update only on new pod generation. But it won't apply on a `k rollout restart`... and maybe this is better that way.